### DISCUSSION TITLE
mac m1 harbor 2.4.0，直接使用安装脚本部署不能用呢

### DISCUSSION THREAD

**Discussion by yuyadeil** [goharbor/harbor#15920] [Q&A] [answered]
mac m1 harbor 2.4.0，直接使用安装脚本部署不能用

<img width="464" alt="截屏2021-11-01 下午11 56 00" src="https://user-images.githubusercontent.com/7915032/139703075-d685bff2-c766-4925-ad6b-ccb18fe5693b.png">

**Comment by wy65701436** [comment, marked as answer]
hi, install Harbor on Mac is not official supported and tested. You can share the harbor.yml and full log.

**Comment by AlexAny** [comment]
harbor-offline-installer-v2.2.4.tgz
harbor-offline-installer-v2.3.5.tgz
harbor-offline-installer-v2.4.1.tgz
all are not ok.

`prepare base dir is set to /usr/local/harbor
WARNING: The requested image's platform (linux/amd64) does not match the detected host platform (linux/arm64/v8) and no specific platform was requested
Traceback (most recent call last):
  File "/usr/src/app/main.py", line 15, in <module>
    cli()
  File "/usr/lib/python3.9/site-packages/click/core.py", line 1137, in __call__
    return self.main(*args, **kwargs)
  File "/usr/lib/python3.9/site-packages/click/core.py", line 1062, in main
    rv = self.invoke(ctx)
  File "/usr/lib/python3.9/site-packages/click/core.py", line 1668, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/usr/lib/python3.9/site-packages/click/core.py", line 1404, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/usr/lib/python3.9/site-packages/click/core.py", line 763, in invoke
    return __callback(*args, **kwargs)
  File "/usr/src/app/commands/prepare.py", line 37, in prepare
    config_dict = parse_yaml_config(conf, with_notary=with_notary, with_trivy=with_trivy, with_chartmuseum=with_chartmuseum)
  File "/usr/src/app/utils/configs.py", line 94, in parse_yaml_config
    with open(config_file_path) as f:
FileNotFoundError: [Errno 2] No such file or directory: '/input/harbor.yml'`

**Comment by yuyadeil** [comment]
目前官方不支持，维护力度不够啊！可以找下别人提供的最新版



发自我的iPhone


------------------ 原始邮件 ------------------
发件人: AlexAny ***@***.***&gt;
发送时间: 2021年12月24日 09:52
收件人: goharbor/harbor ***@***.***&gt;
抄送: yuyadiel ***@***.***&gt;, Manual ***@***.***&gt;
主题: 回复：[goharbor/harbor] mac m1 harbor 2.4.0，直接使用安装脚本部署不能用呢 (Discussion #15920)

**Reply by bigmebala** [reply]
好吧，这么看真的是挺坑，谢谢兄弟,前几天刚放弃了～

**Comment by QiuToo** [comment]
https://blog.csdn.net/qq_49496397/category_11542232.html可以帮到你
